packages/ra-ui-materialui/src/field/ArrayField.spec.js

```diff
@@ -60,9 +61,11 @@ describe('<ArrayField />', () => {
             </ArrayField>
         );
         const wrapper = mount(
-            <Provider store={createStore(() => ({}))}>
-                <Dummy />
-            </Provider>
+            <MemoryRouter>
```

Comment:
you can pretty much use the TestContext here

Reply:
I cleanup those tests and rewrote them using rtl. One of them was useless